aspnetcore/getting-started/index.md

```diff
@@ -35,6 +35,8 @@ Open a command shell, and enter the following command:
 dotnet new webapp -o aspnetcoreapp
 ```
 
+The preceding command creates a new application of type `webapp` for you.  The `-o` parameter creates a directory named *aspnetcoreapp* where your app is stored, and populates it with the required files.
```

Comment:
The preceding command:

* Creates a new web app.
* The `-o` parameter creates a directory named *aspnetcoreapp* with the source files for the app.

Reply:
I don't allow you/your unless it makes the statement more clear. In most cases (like this), it just adds words.

Reply:
`@tdykstra` is on vacation.

Reply:
`@tdykstra` is on .NET now

Reply:
@Rick-Anderson, Thanks for the guidance! Will move to ga, follow "you/your" guideline.  I thought Tom and I might still review each others PR's.  I didn't realize he was on vacation, thanks.  Will set reviews to others on ASP.NET sub-team members going forward.